tedana/workflows/tedana.py

```diff
@@ -178,9 +178,11 @@ def _get_parser():
                           default=False)
     optional.add_argument('--debug',
                           dest='debug',
-                          help=argparse.SUPPRESS,
                           action='store_true',
-                          default=False)
+                          help=('Logs in the terminal will have increased '
+                                'verbosity, and also will also be written  '
+                                'into a .txt file in the output directory.'),
+                          default=True)
```

Comment:
@emdupre @tsalo I thought that `store_true` args could only store `True`; in other words, a default of `True` makes the argument always true. Is that incorrect? In some other code I'm working on that was my experience, but maybe I was handling it incorrectly.

Reply:
Good catch. Yeah, debug's default should be set to False here and in the function.

Reply:
Ah, but then in this setup it won't get written because it requires `--debug` to execute. Okay, this PR will unfortunately need changes.

Reply:
As above, I'm ok if this PR is only to make the debug option visible in the documentation, and if we make the additional changes in a separate PR ! It's up to @benoitberanger how much to tackle, here.
But definitely agree that regardless, this and the other boolean will need to change back to False :+1:

Reply:
Corrected in https://github.com/ME-ICA/tedana/pull/385/commits/e9a5fb1e82d2efe71814f739e3bbcdaa42078ad7